release.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -e
+
+# Usage
+# PUBLISH_GEM=true RELEASE_VERSION=1.2.7 ./release.sh
+
+PHRASE_RUBY_REPO=git@github.com:phrase/phraseapp-ruby.git
+PHRASE_RUBY_TMP=$(mktemp -d /tmp/phraseapp-ruby-lib-XXXX)
+
+trap "rm -Rf $PHRASE_RUBY_TMP" EXIT
+
+if [[ -z "$PHRASE_RUBY_TMP" ]]; then echo "unable to create tmp dir"; exit 1; fi
```

Comment:
we would need to return before the trap is set up. but: I realized the `-z` is not necessary as `set -e` would not continue anyway so you can just remove it again :)